hack/quick_install.sh

```diff
@@ -110,6 +114,11 @@ helm install \
    --set crds.enabled=true
 echo "😀 Successfully installed Cert Manager"
 
+#Install KEDA
+helm repo add kedacore https://kedacore.github.io/charts
+helm install keda kedacore/keda --version ${KEDA_VERSION} --namespace keda --create-namespace --wait
+echo "😀 Successfully installed KEDA"
```

Comment:
Should we add a note that the K8s metrics server is required for cpu and memory e.g. minikube does not have it on by default?